CMakeLists.txt

```diff
@@ -26,8 +26,13 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+option(CLIENT_SIMULATOR "Install the client simulator binary" OFF)
```

Comment:
Please make the client simulator a separate project (in the same way as the skeleton-subsystem). This will remove the need to add an option or any other change to this file.